virtool/references/db.py

```diff
@@ -83,19 +95,25 @@ def __init__(self, app, process_id):
         tracker = self.get_tracker(len(inserted_otu_ids))
 
         for otu_id in inserted_otu_ids:
-            await insert_change(self.db, otu_id, "clone", user_id)
+            await insert_change(self.app, otu_id, "clone", user_id)
             await tracker.add(1)
 
     async def cleanup(self):
         ref_id = self.context["ref_id"]
 
         query = {"reference.id": ref_id}
 
+        diff_file_change_ids = await self.db.history.distinct("_id", {
+            **query,
+            "diff": "file"
+        })
+
         await asyncio.gather(
             self.db.references.delete_one({"_id": ref_id}),
             self.db.history.delete_many(query),
             self.db.otus.delete_many(query),
-            self.db.sequences.delete_many(query)
+            self.db.sequences.delete_many(query),
+            virtool.history.utils.remove_diff_files(diff_file_change_ids)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'id_list' in function call](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=4787087)